doc/design/control-plane.md

```diff
@@ -411,21 +462,22 @@ message EventMessage {
 }
 ```
 
+Read more about eventing [here](./events.md).
 An up to date API of the event format can be fetched
- [here](https://github.com/openebs/mayastor-dependencies/blob/develop/apis/events/protobuf/v1/event.proto).
+ [here](https://github.com/openebs/mayastor-dependencies/blob/HEAD/apis/events/protobuf/v1/event.proto).
 
-## Distributed Tracing
+## Tracing and Telemetry
 
 Tracing means different things at different levels. In this case, we are referring to tracing component boundary tracing.
 
-Tracing is by default implemented using open telemetry and, by default, we have provided a subscriber for jaeger. From jaeger, the information can be
-forwarded to, Elastic Search, Cassandra, Kafka, or whatever. In order to achieve full tracing support, all the gRPC requests and replies should add
-HTTP headers such that we can easily tie them together in whatever tooling is used. This is standard practice but requires a significant amount of work.
-The key reason is to ensure that all requests and responses pass along the headers, from REST to the scheduling pipeline.
+Tracing is by default implemented using open telemetry and, by default, we have provided a subscriber for [Jaeger]. From [Jaeger], the information can be
+forwarded to, Elastic Search, Cassandra, Kafka, etc. In order to achieve full tracing support, all the [gRPC] requests and replies should add
+`HTTP` headers such that we can easily tie them together in whatever tooling is used. This is standard practice but requires a significant amount of work.
+The key reason is to ensure that all requests and responses pass along the headers, from `REST` to the scheduling pipeline.
 
-We also need to support several types of transport and serialization mechanisms. For example, HTTP/1.1 REST requests to HTTP/2 gRCP request to
+We also need to support several types of transport and serialization mechanisms. For example, `HTTP/1.1 REST` requests to `HTTP/2 gRPC` request to
  a KV store operation to etcd. For this, we will use [Tower]. \
-[Tower] provides a not-so-easy to use an abstraction of Request to Response mapping.
+[Tower] provides a not-so-easy to use an abstraction of request to response mapping.
```

Comment:
Nit: Sentence seems incomplete, 'not-so-easy to use...'

Maybe, you mean, not-so-easy framework or interface or tooling?

Reply:
Done